capture/capture_aes.py

```diff
@@ -80,15 +81,30 @@ def setup(cfg: dict, project: Path):
     # Init scope.
     scope_type = cfg["capture"]["scope_select"]
     logger.info(f"Initializing scope {scope_type} ...")
+    # Convert number of cycles into number of samples, if necessary.
```

Comment:
For Husky, we should check that the sampling_rate is indeed an integer multiple of pll_frequency. And print a warning if it's not. Otherwise, the scope is configured with wrong num_samples and offset_samples values.

Reply:
I added a check to make sure that `adc.mul` is an integer